Framework/Core/test/test_OverrideLabels.cxx

```diff
@@ -0,0 +1,93 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#define BOOST_TEST_MODULE Test Framework OverrideLabels
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+
+// We prevent runDataProcessing from starting a workflow
+#define main anything_else_than_main
+#include "Framework/runDataProcessing.h"
+#undef main
+o2::framework::WorkflowSpec defineDataProcessing(o2::framework::ConfigContext const&) { return {}; }
```

Comment:
@ktf I am not very happy about this, but I don't see any other workaround here (I need to test `overrideLabels` in this header). Would you have any other idea for that?

Reply:
We could move most of the header in runDataProcessing.h to runDataProcessing.inc and keep in runDataProcessing.h only the main. I will do it in a separate PR.

Reply:
thank you

Reply:
For the moment let's go ahead with this, though. Since there is a separate PR which would have conflicts.